packages/@storefront/flux-capacitor/src/core/reducers/data/navigations.ts

```diff
@@ -65,21 +67,30 @@ export const receiveNavigations = (state: State, navigations: Store.Navigation[]
 };
 
 // tslint:disable-next-line max-line-length
-export const selectRefinement = (state: State, { navigationId, index: refinementIndex }: Actions.Payload.Navigation.Refinement) => {
-  if (navigationId && refinementIndex != null) {
-    return {
-      ...state,
-      byId: {
-        ...state.byId,
-        [navigationId]: {
-          ...state.byId[navigationId],
-          selected: state.byId[navigationId].selected.concat(refinementIndex),
-        },
+export const selectMultipleRefinements = (state: State, { navigationId, indices }: Actions.Payload.Navigation.MultipleRefinements) => {
+  if (!navigationId) {
```

Comment:
I don't think you need to check this because 1) it must be passed in as a parameter to the action creator and 2) the action creator goes through a validator that checks whether it exists

Reply:
Moved the check into the validators. Resolving conversation.